pkg/reconciler/subscription/subscription_test.go

```diff
@@ -70,7 +70,7 @@ var (
 	serviceURI = "http://" + serviceDNS + "/"
 
 	subscriberGVK = metav1.GroupVersionKind{
-		Group:   "testing.eventing.knative.dev",
+		Group:   "eventing.knative.dev",
```

Comment:
this is testing ducktyping for subscribers. `testing.` was wanted.

Reply:
I changed that because in subscriber validation there is hardcoded logic to validate API group is [`eventing.knative.dev`](https://github.com/knative/eventing/blob/master/pkg/apis/eventing/v1alpha1/subscription_validation.go#L124)

Reply:
ah, thanks.